### ISSUE TITLE
Rework Active Speakers on Landing Page

### ISSUE THREAD

**Issue by marcoroth** [rubyevents/rubyevents#353]
I'm a bit confused by the selection of "Active Speakers" on the landing page. I think it should either be speakers that have given a talk in the last year or so, or speakers that have given a lot of talks but also gave one somewhat recently.

**Comment by adrienpoly** [comment]
Yes it make sense. The only thing is to filter speakers with a GitHub profile

**Comment by marcoroth** [comment]
This was resolved via https://github.com/rubyevents/rubyevents/pull/662
